### DISCUSSION TITLE
ScriptReference/Networking.UnityWebRequest.EscapeURL

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#211] [Notes]
# ScriptReference/Networking.UnityWebRequest.EscapeURL

https://docs.unity3d.com/ScriptReference/Networking.UnityWebRequest.EscapeURL.html

**Comment by mvi** [comment]
```csharp
UnityWebRequest.EscapeURL("https://www.example.com/foo?bar=123!")
```
  
produces  
  
`https%3a%2f%2fwww.example.com%2ffoo%3fbar%3d123%21`
